backend/controllers/searchExternalUsers.test.js

```diff
@@ -0,0 +1,247 @@
+const { searchFactory } = require('./searchExternalUsers')
```

Comment:
Just the search.test.js file renamed, and a few bits changed to match the variables passed to nunjucks